src/progress_bar.rs

```diff
@@ -54,6 +54,11 @@ impl ProgressBar {
         }
     }
 
+    /// Get the a clone of current progress bar style.
+    pub fn current_style(self) -> ProgressStyle {
```

Comment:
Please change the name to `style()`, and return a reference. Then, the caller can clone it.

Reply:
Is this possible when it is behind a mutex?

Reply:
That's a fair point, it is not!